package.json

```diff
@@ -72,6 +72,7 @@
     "allure-commandline": "2.21.0",
     "autoprefixer": "10.4.14",
     "babel-loader": "8.3.0",
+    "body-scroll-lock": "4.0.0-beta.0",
```

Comment:
Is there a reason to use the beta version?

Reply:
Just the latest version available. I notice the package hasn't been updated in a while so may be worth us finding an alternative in the future but we use this already in fozzie with a lot of success :)